worlds/tunic/__init__.py

```diff
@@ -142,13 +148,29 @@ def generate_early(self) -> None:
 
         self.player_location_table = standard_location_name_to_id.copy()
 
+        if self.options.local_fill == -1:
+            if self.options.grass_randomizer:
+                self.options.local_fill.value = 95
+            #     if self.options.breakable_shuffle:
+            #         self.options.local_fill.value = 96
+            #     else:
+            #         self.options.local_fill.value = 95
+            # elif self.options.breakable_shuffle:
+            #     self.options.local_fill.value = 40
```

Comment:
let's remove these so they aren't referenced in the grass PR since it doesn't exist yet

Reply:
I figured they'd make the merge later easier but oh well